webrender/src/device.rs

```diff
@@ -307,9 +313,7 @@ impl VertexDescriptor {
 
         if !self.instance_attributes.is_empty() {
             instance.bind(gl);
-            let instance_stride: u32 = self.instance_attributes
-                                           .iter()
-                                           .map(|attr| attr.size_in_bytes()).sum();
```

Comment:
This assumes we are packing attributes in a way that does not leave any padding. It'd be nice to have a check that the assumption holds true.